yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -55,6 +58,12 @@ func init() {
 	registerTargetDBAsSourceConnFlags(exportDataFromTargetCmd)
 	registerExportDataFlags(exportDataFromTargetCmd)
 	hideExportFlagsInFallForwardOrBackCmds(exportDataFromTargetCmd)
+
+	BoolVar(exportDataFromTargetCmd.Flags(), &uniqueKeyColumnsExists, "unique-key-columns-exists", true,
```

Comment:
Let's rename the flag as `--transaction-ordering`. Even in the presence of uniq keys, a user may decide to go full throttle (i.e. without transaction ordering) if their app never modifies UKs or modifies in such a way that it never causes conflict.
Presence of unique keys in not the root problem, it is the way in which they are modified (if at all).